desc/equilibrium/coords.py

```diff
@@ -113,7 +113,9 @@ def periodic(x):
 
     @functools.partial(jit, static_argnums=1)
     def compute(y, basis):
-        grid = Grid(y, sort=False, jitable=True)
+        # need some dummy spacing values bc iota_num/iota_den still computed using FSA
```

Comment:
this is kind of a kludge. We precompute iota using a good grid for the FSA, but the dependency structure stills tries to use the custom grid to compute iota_num etc. Not sure of a better option?

Reply:
Have you thought of modifying the compute functions such that the quantity is computed only if grid.spacing exists?

(in general, the drawback of patching things via dummy spacing is that things may silently get computed incorrectly).

Reply:
could we just update the `data` with `iota_num` and `iota_den` etc so it knows it exists and does not try to recompute it?

Reply:
@unalmis I think we have to return something from the compute function in both cases to prevent infinite recursion in `compute`. We could return nan if grid.spacing doesn't exist? But that might lead to some weird results down the line and be hard to tell where they're coming from.

@dpanici I think that might work. If we know ahead of time what keys to override we can create dummy values just for those, but again could be a source of weird bugs

Reply:
Maybe worth a try and then see if it works?